deploy/namespace.yaml

```diff
@@ -1,6 +1,7 @@
+{{define "handlerPrefix"}}{{with $prefix := .HandlerPrefix}}{{$prefix | printf "%s-"}}{{end -}}{{end}}
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: {{ .Namespace }}
+  name: {{template "handlerPrefix" .}}{{ .HandlerNamespace }}
```

Comment:
Hummm I was all wrong, namespaces are different between CNAO and operator, we don't need to prefix it, sorry... can we go back to just namespace ?